tests/flow/test_udf.py

```
@@ -0,0 +1,1035 @@
+
+class testUDF():
+    def __init__(self):
+        self.env, self.db = Env()
+        self.graph = self.db.select_graph(GRAPH_ID)
+        self.conn = self.env.getConnection()
+
+    def udf_list(self, lib=None, with_code=None):
+        libs = []
+        res = self.db.udf_list(lib, with_code)
+
+        for l in res:
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)